code/modules/telesci/telepad.dm

```diff
@@ -112,18 +112,20 @@
 		return ..()
 
 /obj/machinery/telepad_cargo/attack_hand(mob/living/user)
-	if (active == TRUE)
-		user << "You switch the receiver off."
-		icon_state = "pad-idle-o"
-		active = FALSE
-		if (cargopads[src] != null)
-			LAZYREMOVE(cargopads, src)
-	else
-		user << "You switch the receiver on."
-		icon_state = "pad-idle"
-		active = TRUE
-		if (cargopads[src] != null)
-			LAZYADD(cargopads, src)
+	if(do_after(user, 20*W.toolspeed, target = src, unique=TRUE))

```

Comment:
also the do_after is not needed since we have logging, and it's shit to do a do_after for something so simple